Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.11
+
+WORKDIR /app
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py"]
```

Comment:
Verify that the path `app/main.py` is correct and that the file exists. Otherwise, the container may fail to start.